components/Tabs/Tabs.js

```diff
@@ -69,4 +74,6 @@ class TabCard {
 - In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter
 
 */
-let tabs = document.querySelectorAll();
\ No newline at end of file
+let tabs = document.querySelectorAll('.tab').forEach(link => new TabLink(link))
+
+console.log(TabLink)
```

Comment:
Good job using console.logs to check your code. Just delete them when you're done so they don't clutter your console!